packages/doc/src/stories/Switch.stories.js

```diff
@@ -0,0 +1,16 @@
+import { MkrSwitch } from '../../../mikado_reborn/src/components';
+
+export default {
+  title: 'Components/Switch',
+  component: MkrSwitch,
+  argTypes: {},
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  data: () => ({ model: false }),
+  template: '<mkr-switch v-bind="$props" v-model="model" />',
+});
+
+export const Switch = Template.bind({});
+Switch.args = {};
```

Comment:
No need to add argTypes or args, you can remove all :)
```suggestion
export default {
  title: 'Components/Switch',
  component: MkrSwitch,
};

const Template = () => ({
  data: () => ({ model: false }),
  template: '<mkr-switch v-bind="$props" v-model="model" />',
});

export const Switch = Template.bind({});
```